docs/src/basic/phasespaces.md

```diff
@@ -30,11 +30,11 @@ t = 200 # how long to evolve each one
 
 ξς, φς, ις = boundarymap(bd, t, n)
 
-using PyPlot # enables plot_boundarymap function
+using PyPlot # enables plot function
 
 colors = ["C$(rand(1:9))" for i in 1:n] # random colors
 
-plot_boundarymap(ξς, φς, ις, color = colors)
+plot(ξς, φς, ις, color = colors)
```

Comment:
Could you please revert the change of `plot_boundarymap`? This function can't be safely extend `plot` because it uses vectors of numbers.